app/komponenter/språkvelger/språkvelger.tsx

```diff
@@ -0,0 +1,38 @@
+import { Select } from '@navikt/ds-react';
+import { useSpråk } from '~/root';
+import { LocaleType } from '~/typer/sanity/sanity';
+import css from './språkvelger.module.css';
+import { GlobeIcon } from '@navikt/aksel-icons';
+
+export const Språkvelger = () => {
+  const [, settSpråk] = useSpråk();
+
+  function handleChange(evt: React.ChangeEvent<HTMLSelectElement>) {
+    settSpråk(evt.target.value as LocaleType);
```

Comment:
Kunne det vært en idé og sette `evt.target.value`  som en variabel? 😄 da kan man gi den et beskrivende navn så man lettere forstår hva verdien egentlig inneholder

Reply:
Smart for lesbarheten, det ble gjort her [969d358](https://github.com/bekk/nav-familie-endringsmelding/pull/19/commits/969d358a939238b1122f9e2ce1ba536c8ec20ec0)